templates/emails/email_base.html

```diff
@@ -0,0 +1,42 @@
+{% load static %}
+{% load inlinecss %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="Content-Type" content="text/html charset=UTF-8"/>
+    <title>Confirm your email address</title>
```

Comment:
I added it mainly for testing in the browser. I can definitley remove this.

Reply:
It seems that having the subject as the title could be useful since some email apps use it for notifications and such. So I've make it something you can specify in the template.